presentations/ffi/slides.adoc

```diff
@@ -125,11 +125,11 @@ See https://doc.rust-lang.org/reference/linkage.html
 include::./rustlib-Cargo.toml[]
 ----
 
-////
+--
```

Comment:
The `[.notes]` marker seems to be missing here.

Reply:
wow i missed so many. :see_no_evil: thank you so much

Reply:
so odd i pushed my branch but github does not show this outdated :ugh